opendbc/car/honda/values.py

```diff
@@ -104,6 +104,9 @@ class Footnote(Enum):
   CIVIC_DIESEL = CarFootnote(
     "2019 Honda Civic 1.6L Diesel Sedan does not have ALC below 12mph.",
     Column.FSR_STEERING)
+  OP_STEER_TO_ZERO = CarFootnote(
+    "No ALC below 0 mph when openpilot Longitudinal Control is enabled. Safety features such as AEB and FCW will be disabled.",
+    Column.FSR_STEERING)
```

Comment:
Not sure what you meant here, but "No ALC below 0 mph" needs fixing or clarification.